sys/contrib/ipfilter/netinet/ip_fil.h

```diff
@@ -1321,7 +1321,7 @@ typedef	int	(*ipftq_delete_fn_t)(struct ipf_main_softc_s *, void *);
 typedef	struct	ipfobj	{
 	u_32_t		ipfo_rev;	/* IPFilter version number */
 	u_32_t		ipfo_size;	/* size of object at ipfo_ptr */
-	void		*ipfo_ptr;	/* pointer to object */
+	void * __kerncap ipfo_ptr;	/* pointer to object */
```

Comment:
nit. keep indentation consistent with the rest of the file
```suggestion
	void			* __kerncap ipfo_ptr;	/* pointer to object */
```

Reply:
@brooksdavis has previously stated he prefers this version to avoid reindenting the rest of the line as much (e.g. the comment in this case still lines up with the other comments)